garage/tf/algos/npo.py

```diff
@@ -406,7 +406,7 @@ def _build_entropy_term(self, i):
                 policy_dist_info_flat = self.policy.dist_info_sym(
                     i.flat.obs_var,
                     i.flat.policy_state_info_vars,
-                    name="policy_dist_info_flat")
+                    name="policy_dist_info_flat_entropy")
```

Comment:
Why do you change the name? And the name is inappropriate.

Reply:
I explained above. Basically the model design does not allow us to call `dist_info_sym` with the same name twice.

Reply:
Is it actually pointing to the same `policy_dist_info_flat` as in `_build_policy_loss`?